test/custom-elements/index.ts

```diff
@@ -138,6 +123,27 @@ describe('custom-elements', function() {
 					})), expected_warnings);
 				}
 			}
+
+			// NOTE: Chromium may exit due to SIGSEGV, so retry in that case.
+			let count = 0;
+			do {
+				count++;
+				try {
+					const page = await getNewPage(browser);
+					await page.goto('http://localhost:6789');
+					const result = await page.evaluate(() => test(document.querySelector('main')));
+					if (result) console.log(result);
+					assertWarnings();
+					break;
+				} catch (err) {
+					if (count === 5 || browser.isConnected()) {
+						console.log(addLineNumbers(code));
+						assertWarnings();
+						throw err;
+					}
+					browser = await launchPuppeteer();
+				}
+			} while (count <= 5);
```

Comment:
I feel like this has the potential to be neater to avoid the double `assertWarnings` and `5` number. But I'm not sure how either, so looks good to me 😬

Reply:
I think it's fine because this code is modified very infrequently.😆